android/src/main/java/com/it_nomads/fluttersecurestorage/ciphers/RSACipher18Implementation.java

```diff
@@ -189,30 +189,13 @@ private void createKeys(Context context) throws Exception {
                         .setCertificateNotBefore(start.getTime())
                         .setCertificateNotAfter(end.getTime());
 
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-                    builder.setIsStrongBoxBacked(true);
-                }
-
                 spec = builder.build();
             }
-            try {
-                Log.i("fluttersecurestorage", "Initializing");
-                kpGenerator.initialize(spec);
-                Log.i("fluttersecurestorage", "Generating key pair");
-                kpGenerator.generateKeyPair();
-            } catch (StrongBoxUnavailableException se) {
-                spec = new KeyGenParameterSpec.Builder(KEY_ALIAS, KeyProperties.PURPOSE_DECRYPT | KeyProperties.PURPOSE_ENCRYPT)
-                        .setCertificateSubject(new X500Principal("CN=" + KEY_ALIAS))
-                        .setDigests(KeyProperties.DIGEST_SHA256)
-                        .setBlockModes(KeyProperties.BLOCK_MODE_ECB)
-                        .setEncryptionPaddings(KeyProperties.ENCRYPTION_PADDING_RSA_PKCS1)
-                        .setCertificateSerialNumber(BigInteger.valueOf(1))
-                        .setCertificateNotBefore(start.getTime())
-                        .setCertificateNotAfter(end.getTime())
-                        .build();
-                kpGenerator.initialize(spec);
-                kpGenerator.generateKeyPair();
-            }
+            
+            Log.i("fluttersecurestorage", "Initializing");
```

Comment:
Could you please remove logs ?

Reply:
I did not add these log statements (they just moved), so felt unconformtable removing it. You sure?

Reply:
It's fine, we are getting rid of them for now.